src/plugins/template/backend/CMakeLists.txt

```diff
@@ -40,7 +40,9 @@ target_compile_definitions(interpreter_backend
 )
 target_link_libraries(interpreter_backend PRIVATE ngraph::builder ngraph::reference openvino::util openvino::runtime::dev)
 
-target_include_directories(interpreter_backend PUBLIC $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>)
+target_include_directories(interpreter_backend PUBLIC $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>
+                           ${CMAKE_CURRENT_SOURCE_DIR}/../../..
+                           ${CMAKE_CURRENT_SOURCE_DIR}/../../../common/conditional_compilation/include)
```

Comment:
remove

Reply:
You can just link with cc library instead of this include directories